shell/src/main/java/alluxio/shell/command/ChownCommand.java

```diff
@@ -53,6 +55,11 @@ public Options getOptions() {
     return new Options().addOption(RECURSIVE_OPTION);
   }
 
+  private static final Pattern USER_PATTERN = Pattern.compile("(?<user>[a-z_][a-z0-9_-]*)");
+
+  private static final Pattern USER_GROUP_PATTERN =
+      Pattern.compile("(?<user>[a-z_][a-z0-9_-]*):(?<group>[a-z_][a-z0-9_-]*)");
```

Comment:
for `[a-z0-9_-]` , do we only allow lower-case characters? I would use `\w`
https://docs.oracle.com/javase/tutorial/essential/regex/pre_char_classes.html